File: Beecrowd-Problems/C#/done!/problem1006.cs
using System;
using System.Globalization;

class URI
{

    static void Main(string[] args)
    {
        CultureInfo.CurrentCulture = CultureInfo.InvariantCulture;
        CultureInfo.CurrentUICulture = CultureInfo.InvariantCulture;
        double weightA = 2d, weightB = 3d, weightC = 5d;
        double A = double.Parse(Console.ReadLine());
        double B = double.Parse(Console.ReadLine());
        double C = double.Parse(Console.ReadLine());
        double average = (weightA * A + weightB * B + weightC * C) / (weightA + weightB + weightC);
        Console.WriteLine($"MEDIA = {average:F1}");
    }
}

File: Beecrowd-Problems/Python/done!/problem1006.py
weight_a = 2
weight_b = 3
weight_c = 5
a = float(input())
b = float(input())
c = float(input())

average = (weight_a * a + weight_b * b + weight_c * c) / (weight_a + weight_b + weight_c)

print("MEDIA = {:.1f}".format(average))